engine/runtime-integration-tests/src/test/java/org/enso/compiler/ExecStrictCompilerTest.java

```diff
@@ -55,17 +55,20 @@ public static void closeEnsoContext() throws Exception {
 
   @Test
   public void redefinedArgument() throws Exception {
-    var module = ctx.eval("enso", """
-    type My_Type
-        Value a b c a
-    """);
     try {
+      var module = ctx.eval("enso", """
+      type My_Type
+          Value a b c a
+      """);
       var run = module.invokeMember("eval_expression", "My_Type.Value");
```

Comment:
Is this line even reachable? Shouldn't it be deleted and replaced by the `fail("Expecting no returned value: " + run);` line? I though that the compilation exception should be thrown from `ctx.eval` now, and now from `module.invokeMember("eval_expression")`?

Reply:
5b7038f01a